draft-birkholz-rats-corim.md

```diff
@@ -99,6 +99,52 @@ See {{-coswid}}.
 
 Privacy Considerations
 
+# Concise SWID Data Definition
+
+# 2.7 The link-entry Map
+
+The link-entry map has been extended to define new relationship between a CoSWID and
+CoMID(identifies a hardware/firmware module).
+
+When defining CoSWID to CoMID link entry map the following optional elements
+should not be used:
+ artifact
+ ownership
+ use
+
+# CoSWID Indexed Label Values
+
+# 4.4. Link Rel Values
+New indexes in the range 12 to 15 will be registered in the
+IANA "Software Tag Link Relationship Values" registry defined in Section 6.2.7.
+
+ | Index | Relationship Type Name | Definition                                     |
+ --------|----------------------- |--------------------------------------------|
+ | 12    | m-requires             | The link references a prerequisite module  |
+ |       |                        | tag(s) for installing this software.       |
+ | 13    | m-runs-on              | The link references a module tag that      |
+ |       |                        | this software run on                       |
+ | 14    | m-part-of              | The link references a module tag that this |
+ |       |                        | software is part of                        |
+ | 15    | m-supplemental         | The link references a module tag that the  |
+ |       |                        | referencing tag supplements.               |
+
```

Comment:
This table should go in the IANA section below.  This section should, for each new relation defined, provide the semantic details.

Reply:
This is how it is documented in CoSWID specification in an exact manner! Hence I replicated the tabular syntax from
Section 4.4 of CoSWID specification!